recipes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "meson" %}
+{% set version = "0.45.0" %}
+{% set sha256 = "d7447bbb232712de303135f23e8f41ef767d02d50d3022cf055d13f69c2540be" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I guess we should use `python -m pip install --no-deps --ignore-installed` nowadays.

Reply:
Sure I'll change that. What is the advantage of it? `pip install` goes through `setup.py install`, too and is more likely to crash if something is missing or am I wrong?

Reply:
https://github.com/conda-forge/conda-forge.github.io/pull/518